### PR TITLE
Fix disorder of request.

### PR DESCRIPTION
**Pull request by zhengrenjie**
Guarantee that the requests within one channel is always in order.

### CODE REVIEW

File: test/src/test/java/com/distkv/drpc/test/protocol/WorkerPoolTest.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          .newBuilder()
+          .setValue(i)
+          .build();
+
+      service.put(putRequest);
+
+      try {
+        EchoProtocol.GetResponse response = service.get().get();
+        if (response.getValue() != i) {
+          Assert.fail("Response error: " + response.getValue() + " expect: " + i);
```
Comment (jovany-wang) [inline test/src/test/java/com/distkv/drpc/test/protocol/WorkerPoolTest.java]: Assert.assertEquals(response.getValue(), i);

**Review by jovany-wang — CHANGES_REQUESTED** [review]

File: test/src/test/java/com/distkv/drpc/test/protocol/WorkerPoolTest.java (hunk 1 above)
Comment (zhengrenjie) [reply test/src/test/java/com/distkv/drpc/test/protocol/WorkerPoolTest.java]: Fixed!

**Review by kairbon — APPROVED** [review]
Good job, I learned a lot from this PR.

**Review by jovany-wang — COMMENTED** [review]
@kairbon I think you should review this one more time

**Review by jovany-wang — APPROVED** [review]

**Pull request merged**